spec/game_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require_relative '../lib/game.rb'
+
+RSpec.describe Game do
+  let(:game) { Game.new }
+  let(:players) { game.players('Amanda', 'Charli') }
+
+  describe '#players' do
+    it 'should return the second player witch is the current one' do
+      expect(players.player).to eql('Charli')
+    end
+  end
+
+  describe '#win?' do
+    it 'should return true when the first row of the board is filled with the same symbol' do
+      game.board_game.board = %w[O O O 4 5 6 7 8 9]
+      expect(game.win?).to eql(true)
+    end
+    it 'should return true when the second row of the board is filled with the same symbol' do
```

Comment:
kindly leave an empty line between `it` block, check an example below;
#Bad 
```
 it 'should return true when the third row of the board is filled with the same symbol' do
      game.board_game.board = %w[1 2 3 4 5 6 O O O]
      expect(game.win?).to eql(true)    
 end
  it 'should return true when the first column of the board is filled with the same symbol' do
      game.board_game.board = %w[O 2 3 O 5 6 O 8 9]
      expect(game.win?).to eql(true)
  end
```

#Good
```
 it 'should return true when the third row of the board is filled with the same symbol' do
      game.board_game.board = %w[1 2 3 4 5 6 O O O]
      expect(game.win?).to eql(true)
  end

  it 'should return true when the first column of the board is filled with the same symbol' do
      game.board_game.board = %w[O 2 3 O 5 6 O 8 9]
      expect(game.win?).to eql(true)
  end
```
Kindly apply for the rest of the `it` blocks and go ahead and merge...
check the [link](https://github.com/rubocop-hq/rspec-style-guide#empty-lines-between-describes) for more info........